packages/livekit-rtc/package.json

```diff
@@ -56,9 +65,9 @@
   },
   "scripts": {
     "prebuild": "node -p \"'export const SDK_VERSION = ' + JSON.stringify(require('./package.json').version) + ';'\" > src/version.ts",
-    "build:tsc": "pnpm prebuild && tsc && cp -r src/napi dist/",
-    "build": "pnpm build:tsc && napi build --platform --release --dts native.d.ts --js native.cjs --pipe \"prettier -w\" src/napi",
-    "artifacts": "pnpm build:tsc && napi artifacts",
+    "build:ts": "pnpm prebuild && tsup --onSuccess \"tsc --declaration --emitDeclarationOnly\" && cp -r src/napi dist/ && cp -r src/napi/* dist/",
```

Comment:
this line is silly but it's to get around esbuild's issues with dynamic import of fs

Reply:
is the order of things correct here? it looks like it first copies napi files but napi build is only called afterwards in the `build` script 👀

Reply:
hm it was like this also before, just not sure I understand the reason 🤷

Reply:
there's a few files in src/napi already pre-generation, namely the shim to translate them from CJS to ESM if needed. i do think this might be a bit more convoluted than necessary but it works and doesn't seem to take too long on this part